### PR TITLE
Adding CoinSwitch exchange

### PR DESCRIPTION
**Pull request by vimalsagar**
Adding CoinSwitch Exchange to buy/exchange ETH and send it to Metamask wallet.

### CODE REVIEW

**Comment by nandubatchu** [comment]
@alextsg Can you please takeout some time and review this PR.

**Comment by kumavis** [comment]
rating here seems good https://www.trustpilot.com/review/coinswitch.co

**Comment by kumavis** [comment]
Also supports USD via an indacoin integration (uk company) https://indacoin.com/

**Comment by kumavis** [comment]
@vimalsagar sorry for the long delay in review

File: ui/app/components/modals/deposit-ether-modal.js [hunk 1]
```diff
@@ -29,6 +31,9 @@ function mapDispatchToProps (dispatch) {
     toCoinbase: (address) => {
       dispatch(actions.buyEth({ network: '1', address, amount: 0 }))
     },
+    toCoinSwitch: (address) => {
+      dispatch(actions.buyEth({ network: '5', address, amount: 0 }))
```
Comment (kumavis) [inline ui/app/components/modals/deposit-ether-modal.js]: `network` indicates the ethereum `networkId` ( mainnet vs testnets ). I suggest adding a new field like `service: 'coinswitch'` and using that to generate the url in `buy-eth-url.js`

**Review by kumavis — CHANGES_REQUESTED** [review]

File: ui/app/components/modals/deposit-ether-modal.js (hunk 1 above)
Comment (vimalsagar) [reply ui/app/components/modals/deposit-ether-modal.js]: @kumavis I have made the changes. Please have a look.

**Review by kumavis — DISMISSED** [review]

**Comment by kumavis** [comment]
whoops i broke everything, hold up

**Review by kumavis — DISMISSED** [review]

**Review by kumavis — APPROVED** [review]

**Comment by kumavis** [comment]
thanks @vimalsagar

**Pull request merged**